cypress/integration/buy-shirt.spec.ts

```diff
@@ -0,0 +1,19 @@
+describe("Buy a t-shirt", () => {
+  it("then the t-shirt should be bought", () => {
+    cy.visit("http://automationpractice.com/");
+    cy.get(".sf-menu > :nth-child(3) > a").click();
+    cy.get(".ajax_add_to_cart_button > span").click();
+    cy.get(".button-container > .button-medium > span").click();
+    cy.get(".cart_navigation > .button > span").click();
+    cy.get("#email").type("aperdomobo@gmail.com");
+    cy.get("#passwd").type("WorkshopProtractor");
+    cy.get("#SubmitLogin > span").click();
+    cy.get(".cart_navigation > .button > span").click();
+    cy.get("#cgv").click();
+    cy.get(".cart_navigation > .button > span").click();
+    cy.get(".bankwire").click();
+    cy.get("#cart_navigation > .button > span").click();
+    cy.get(".cheque-indent > .dark").should("have.text", "Your order on My Store is complete.");
+    cy.screenshot("Buy a t-shirt");
```

Comment:
@JenniferCastiblanco That line is not required, what i meant is to add the screenshot of the test passed as a comment, in that way we can validate the execution of the test

Reply:
Something like this
![Screen Shot 2022-05-09 at 4 31 13 PM](https://user-images.githubusercontent.com/36927617/167502000-a2ff0a98-c423-41cc-8b94-31df1410bb0e.png)

Reply:
![Screenshot_1](https://user-images.githubusercontent.com/82177981/167507707-12718dd0-9031-4166-9c6e-65141926f1b1.png)